hazelcast/src/main/java/com/hazelcast/cp/internal/raft/impl/state/FollowerState.java

```diff
@@ -46,6 +53,8 @@ public class FollowerState {
 
     private long appendRequestAckTimestamp;
 
+    private long flowControlSequenceNumber;
```

Comment:
minor: 
```suggestion
    private long correlationSequenceNumber;
```

Reply:
I think it is more terminologically correct to leave it as it is, since the sequence number is used for flow control.